.github/workflows/main.yml

```diff
@@ -390,12 +388,15 @@ jobs:
         run: make lint-docker
       - name: Run the Shellcheck linter
         run: make lint-shell
-      - name: Check for markdown linting errors
+      - name: Run the Markdown linter
         run: make lint-markdown
       - name: Run the GitHub Actions linter
         run: make lint-actions
       - name: Check for warnings in documentation
         run: make build-docs
+      # Run after the docs build
```

Comment:
If you're going to leave a comment here, please call out why this matters.

Reply:
👍 Changed in [f996a00](https://github.com/PrairieLearn/PrairieLearn/pull/11472/commits/f996a00ad3991f477f6fa2b403ae03f618fa6b6f).